src/context/TxContext/index.tsx

```diff
@@ -76,28 +82,61 @@ function TxContextProvider({children}: PropTypes): React.ReactElement {
     const args = meta?.args.map(({name}) => name).join(', ') || '';
     const title = `Sending transaction ${section}.${method}(${args})`;
     const description = formatCallMeta(meta);
+    const keyringPair = getPair(address);
+    let signerFn;
+
+    const showPreview = async (txPayload: SignerPayloadJSON) => {
+      const info = await transaction.paymentInfo(address);
+      dispatch({
+        type: 'SHOW_TX_PREVIEW',
+        payload: {
+          txPayload: txPayload,
+          params,
+          title,
+          description,
+          partialFee: info.partialFee.toNumber(),
+          isExternalAccount: keyringPair === undefined,
```

Comment:
@ishrak-imam is this still true after your changes? Or we will get a `pair` also for external addresses?

Reply:
oh, the changes are already merged.. I'll rebase and take a look